internal/uvm/create_lcow.go

```diff
@@ -64,6 +64,18 @@ type OptionsLCOW struct {
 	PreferredRootFSType   PreferredRootFSType // If `KernelFile` is `InitrdFile` use `PreferredRootFSTypeInitRd`. If `KernelFile` is `VhdFile` use `PreferredRootFSTypeVHD`
 }
 
+// defaultLCOWOSBootFilesPath returns the default path used to locate the LCOW
+// OS kernel and root FS files. This default is the subdirectory
+// `LinuxBootFiles` in the directory of the executable that started the current
+// process; or, if it does not exist, `%ProgramFiles%\Linux Containers`.
+func defaultLCOWOSBootFilesPath() string {
+	localDirPath := filepath.Join(filepath.Dir(os.Args[0]), "LinuxBootFiles")
```

Comment:
Don't want to rathole even more on this 😇 , but should we be looking for the files in the path as well rather than just the existence of the directory?

Reply:
I want it to be clear where the files are coming from: either the exe directory has "LinuxBootFiles", or they're from the global location. Searching path seems like it could add a lot of confusion when trying to make sure you're using the right version of the files.

Reply:
No we want the directory. We verify the files exist later: https://github.com/Microsoft/hcsshim/blob/cc9f7996884d788e4bc324e7547ec1095c82965a/internal/uvm/create_lcow.go#L161

Reply:
@jhowardmsft think I misread your comment, thought you meant searching %PATH% for the files as well.

Reply:
Ah, no not %PATH%. I really did mean only use `LinuxBootFiles` if it is present as a directory, and there are correct files under there. Otherwise fall back to Linux Containers. Currently, the presence of an empty "LinuxBootFiles" directory would fail.

Reply:
I think it's okay to error if you have an empty LinuxBootFiles directory. I'd rather keep the logic simple. The caller of `NewDefaultOptionsLCOW` can also change the file names searched for after the fact, and this change would break that.